docs/modules/globalfee/README.md

```diff
@@ -0,0 +1,157 @@
+- If the `min_gas_prices` set a denom that is not global fees's denom set. This min_gas_prices denom will not be considered when paying fees.
+- If the `min_gas_prices` have a denom in global fees's denom set, and the  min_gas_prices are lower than global fees, the fee still need to meet the global fees.
+- If the `min_gas_prices` have a denom in global fees's denom set, and the  min_gas_prices are higher than global fees, the fee need to meet the min_gas_prices.
+
+### Fee Checks
+Global fees, min_gas_prices and the paid fees all allow zero coins setup. After parsing the fee coins, zero coins and paid fees will be removed from the min_gas_prices and paid fees. 
+
+Only global fees might contain zero coins, which is used to define the allowed denoms of paid fees.
+
+The [Fee AnteHandle](../../../ante/fee.go) will take global fees and min_gas_prices and merge them into one [combined `sdk.Deccoins`](https://github.com/cosmos/gaia/blob/f2be720353a969b6362feff369218eb9056a60b9/ante/fee.go#L79) according to the denoms and amounts of global fees and min_gas_prices.
+
+If the paid fee is a subset of the combined fees set and the paid fee amount is greater than or equal to the required fees amount, the transaction can pass the fee check, otherwise an error will occur.
+
+### Bypass Fees Message Types
+The above `global fee` and `min_as_prices` fee checks do not apply to bypass message types. Transactions of  bypass message types are free of fee charge. However, if the bypass type transactions still carry nonzero fees, the denom has to be a subset of denoms that global fees defined.
+
+A node can set up its own bypass message types by adding the configuration parameter `bypass-min-fee-msg-types` in `config/app.toml` file.
+
+An example:
+```shell
+
+###############################################################################
+###                        Custom Gaia Configuration                        ###
+###############################################################################
+# bypass-min-fee-msg-types defines custom message types the operator may set that
+# will bypass minimum fee checks during CheckTx.
+#
+# Example:
+# ["/ibc.core.channel.v1.MsgRecvPacket", "/ibc.core.channel.v1.MsgAcknowledgement", ...]
+bypass-min-fee-msg-types = ["/ibc.core.channel.v1.MsgRecvPacket", "/ibc.core.channel.v1.MsgAcknowledgement","/ibc.applications.transfer.v1.MsgTransfer"]
+```
+
+[comment]: <> (Even though each node can set its own `min_gas_prices` and `bypass-min-fee-msg-types`, when the transactions enters a validator's mempool, the transactions carried fees have to satisfy the validator's `min_gas_prices` and `bypass-min-fee-msg-types`'s requirement in order for the validators to process the transacton.)
```

Comment:
@okwme can you confirm if this comment is correct?

Reply:
yes this is correct!